docs/fundamentals/actors/index.md

```diff
@@ -7,30 +7,45 @@ sidebar_position: 0
 type: "docs"
 ---
 
-If you're already familiar with the computer science concepts of the [actor model](https://en.wikipedia.org/wiki/Actor_model) and have experienced implementations in other projects or tools, feel free to [skip to the next section](./wc_actors).
-
-In the actor model, the **actor** is the primitive building block of concurrent computation. Actors receive messages, perform work, send messages, and respond to messages. This model has been around since **1973** and is proof that some designs can be meaningful and applicable today, no matter how old they are.
-
-It's important to note that there are multiple perspectives when it comes to the actor model. There is a more rigid, academic definition and then there are the practical concerns of building real world applications that tend to stretch or modify that definition.
+If you're already familiar with the computer science of the
+[actor model](https://en.wikipedia.org/wiki/Actor_model)
+and have experience implementing actors in other projects or tools,
+feel free to [skip to the next section](./wc_actors).
+
+The actor model is a model of concurrent computation that's been around since
+**1973** and is as meaningful and applicable today than it was 50 years ago.
+In this model, the **actor** is the primitive building block
+of concurrent computation.
+Actors are "black-box" computational entities that can only communicate with
+other actors by passing messages and creating new actors.
```

Comment:
Just to confirm, is this line break change intended to lay these out on different lines or is it just editor word wrap configuration? Word wrap is fine, markdown ignores single line breaks